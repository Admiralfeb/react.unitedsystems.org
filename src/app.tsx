import { ThemeProvider, CssBaseline } from '@material-ui/core';
import { darkTheme } from './theme';
import { createBrowserHistory } from 'history';

import { Router } from 'react-router-dom';
import { NavbarComponent } from 'components/navbar';
import { AppRoutes } from 'routes/app.routes';
import './app.css';
import { RealmAppProvider, RealmApolloProvider } from 'providers';
import { SnackbarProvider } from 'notistack';

const realmID: string = 'usc-tbmbi';

function App() {
  const history = createBrowserHistory({ basename: process.env.PUBLIC_URL });

  return (
    <RealmAppProvider appId={realmID}>
      <RealmApolloProvider>
        <SnackbarProvider maxSnack={3}>
          <ThemeProvider theme={darkTheme}>
            <CssBaseline />
            <div className="content">
              <Router history={history}>
                <NavbarComponent />
                <AppRoutes />
              </Router>
            </div>
          </ThemeProvider>
        </SnackbarProvider>
      </RealmApolloProvider>
    </RealmAppProvider>
  );
}

export default App;
