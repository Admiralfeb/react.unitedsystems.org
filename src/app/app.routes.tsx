import { Switch, Route, Redirect } from 'react-router-dom';
import { Home, Join, NotFound } from './components';
import { Information } from './information/information.routes';

export const AppRoutes = () => {
  return (
    <Switch>
      <Redirect from='/' to='/home' exact />
      <Route path='/home' exact>
        <Home />
      </Route>
      <Route path='/join'>
        <Join />
      </Route>
      <Route path='/information'>
        <Information />
      </Route>
      <Route path='*'>
        <NotFound />
      </Route>
    </Switch>
  );
};
