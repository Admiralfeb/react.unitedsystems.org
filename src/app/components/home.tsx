import { makeStyles, Paper, Typography } from '@material-ui/core';
import React from 'react';

const useStyles = makeStyles({
  root: {},
  header: {
    textAlign: 'center',
  },
  paper: {
    textAlign: 'center',
    width: '80%',
    margin: 'auto',
    padding: 5,
    paddingBottom: 10,
    marginBottom: 5,
  },
  latin: {
    display: 'flex',
    flexDirection: 'column',
    textAlign: 'center',
    border: '1px solid white',
    borderRadius: 10,
    width: 300,
    margin: 'auto',
    marginBottom: 10,
  },
  cursive: {
    fontFamily: 'Brush Script MT, Brush Script Std, cursive',
  },
  subtitle: {
    flex: '0 0',
  },
  video: {
    width: '100%',
    padding: 0,
  },
});

export const Home = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Typography variant='h3' className={classes.header}>
        United Systems Cooperative
      </Typography>
      <Paper className={classes.paper}>
        <div className={classes.latin}>
          <Typography className={classes.cursive} variant='h4'>
            Ad Astra Per Aspera
          </Typography>
          <Typography variant='subtitle2' className={classes.subtitle}>
            Through Hardships to the Stars
          </Typography>
        </div>
        <Typography>
          Established in 3306, the United Systems Cooperative is a tight-knit
          organization of independent pilots operating across the galaxy, both
          in human space, and the uncharted sectors beyond. Little is known
          about this groups motives and plans, though many conflicting reports
          are offered by systems across inhabited space. The one thing the
          reports agree on is that members of this group are fiercely loyal to
          one another, and to the organization as a whole. The United Systems
          Cooperative seeks to create a family where everyone is welcome, no
          matter their background, their allegiances, etc. To create a family
          where everyone can get the help they need, make lasting friendships,
          and have new brothers and sisters to fly with them through the black!
        </Typography>
        <video
          controls
          className={classes.video}
          poster='https://dl.dropbox.com/s/nws07775923ta6z/vidImg.png?dl=0'>
          <source
            src='https://dl.dropbox.com/s/h0n88hy5pgtf06n/about_us.mp4?dl=0'
            type='video/mp4'
          />
          "Your browser does not support this video"
        </video>
      </Paper>
    </div>
  );
};
