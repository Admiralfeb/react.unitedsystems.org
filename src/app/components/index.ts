export * from './about';
export * from './information/information';
export * from './home';
export * from './join';
export * from './notFound';
