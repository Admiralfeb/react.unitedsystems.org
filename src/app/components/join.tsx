import {
  Link,
  makeStyles,
  Paper,
  Typography,
  useMediaQuery,
} from '@material-ui/core';
import { NavLink } from 'react-router-dom';

const useStyles = makeStyles({
  root: {
    height: '100%',
  },
  header: {
    textAlign: 'center',
  },
  paper: {
    textAlign: 'center',
    width: '80%',
    margin: 'auto',
    padding: 5,
    paddingBottom: 10,
    marginBottom: 5,
  },
  iframe: {
    width: '100%',
    height: '59vw',
  },
  step: {
    display: 'grid',
    gridTemplate: '1fr / 1fr 1fr',
  },
  right: {
    textAlign: 'right',
  },
  left: {
    textAlign: 'left',
  },
});

export const Join = () => {
  const classes = useStyles();
  const mediaQuery = useMediaQuery('(min-width:950px)');
  return (
    <div className={classes.root}>
      <Typography variant='h2' className={classes.header}>
        Join Us!
      </Typography>
      <Paper className={classes.paper}>
        <Typography>
          Step 1: Read the rules on the{' '}
          <Link to='/about' target='_blank' component={NavLink}>
            About Page
          </Link>
          .
        </Typography>
        {mediaQuery ? (
          <Typography>Step 2: Fill out the form below</Typography>
        ) : (
          <Typography>
            Step 2: Fill out the{' '}
            <Link
              href='https://docs.google.com/forms/d/e/1FAIpQLSfw7vIL8FY8ZGTF8YFjn3x-zGdhP5k9J6JALXAjXA7_Mm3-Rw/viewform?usp=sf_link'
              target='_blank'>
              USC Join Form
            </Link>
          </Typography>
        )}

        <Typography>
          Step 3: Join our{' '}
          <Link href='https://discord.gg/br8Fy2M6cv' target='_blank'>
            Discord
          </Link>
        </Typography>

        <Typography>
          Recommended: Find us in-game! (UCPC for PC | UCXB for Xbox | UCPS for
          PlayStation)
        </Typography>
        <Typography>
          Recommended: Join our{' '}
          <Link href='https://inara.cz/squadron/7028/' target='_blank'>
            Inara Squadron
          </Link>
        </Typography>
        {mediaQuery && (
          <iframe
            className={classes.iframe}
            title='USC Application Form'
            id='joinFormContainer'
            src='https://docs.google.com/forms/d/e/1FAIpQLSfw7vIL8FY8ZGTF8YFjn3x-zGdhP5k9J6JALXAjXA7_Mm3-Rw/viewform?embedded=true'>
            Loadingâ€¦
          </iframe>
        )}
        {/* <iframe
          className={classes.iframe}
          title='Discord Widget'
          id='embeddedDiscord'
          src='https://discord.com/widget?id=662439414152167434&amp;theme=dark'
          allowTransparency
          frameBorder='0'
          sandbox='allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts'></iframe> */}
      </Paper>
    </div>
  );
};
