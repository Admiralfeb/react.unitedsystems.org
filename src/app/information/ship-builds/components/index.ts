export * from './buildItem';
export * from './builds';
export * from './query';
export * from './selection';
