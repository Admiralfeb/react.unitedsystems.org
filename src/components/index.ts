export * from './home';
export * from '../routes/information.routes';
export * from './join';
export * from './joinForm';
export * from './notFound';
