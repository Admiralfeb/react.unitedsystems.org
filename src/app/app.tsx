import React from 'react';

import { ThemeProvider, CssBaseline } from '@material-ui/core';
import { darkTheme } from './theme';
import { createBrowserHistory } from 'history';

import { Router } from 'react-router-dom';
import { NavbarComponent } from './navbar';
import { AppRoutes } from './app.routes';
import './app.css';
import { RealmAppProvider } from './providers/realmAppProvider';
import { RealmApolloProvider } from './providers/realmApolloProvider';

const realmID: string = 'usc-tbmbi';

function App() {
  const history = createBrowserHistory({ basename: process.env.PUBLIC_URL });

  return (
    <RealmAppProvider appId={realmID}>
      <RealmApolloProvider>
        <ThemeProvider theme={darkTheme}>
          <CssBaseline />
          <div className='content'>
            <Router history={history}>
              <NavbarComponent />
              <AppRoutes />
            </Router>
          </div>
        </ThemeProvider>
      </RealmApolloProvider>
    </RealmAppProvider>
  );
}

export default App;
