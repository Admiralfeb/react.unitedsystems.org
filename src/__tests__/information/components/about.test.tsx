import { cleanup, render, RenderResult } from '@testing-library/react';
import { About } from '../../../app/components';

let componentBody: RenderResult;

describe('About', () => {
  beforeEach(() => {
    componentBody = render(<About />);
  });

  afterEach(() => {
    cleanup();
  });

  it(`should match the snapshot`, () => {
    const { baseElement } = componentBody;
    expect(baseElement).toMatchSnapshot();
  });
});
