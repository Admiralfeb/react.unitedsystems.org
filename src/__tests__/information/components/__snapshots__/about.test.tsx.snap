// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match the snapshot 1`] = `
<body>
  <div>
    <div
      class="makeStyles-root-1"
    >
      <h2
        class="MuiTypography-root makeStyles-header-2 MuiTypography-h2"
      >
        About USC
      </h2>
      <div
        class="MuiPaper-root makeStyles-paper-3 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="makeStyles-header-6"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3"
          >
            Rules
          </h3>
          <div
            class="MuiPaper-root makeStyles-root-7 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="makeStyles-rules-8"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4"
              >
                Discord
              </h4>
              <ul
                class="MuiList-root MuiList-padding"
              >
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      1) Be Respectful
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      2) Be Mature
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      3) Do not @everyone
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      4) No porn or nudity
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      5) No harassment
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      6) No excessive swearing
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      7) No derogatory terms or phrases
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      8) No political discussions as they can become heated.
                    </span>
                  </div>
                </li>
              </ul>
            </div>
            <div
              class="makeStyles-rules-8"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4"
              >
                Members of USC
              </h4>
              <ul
                class="MuiList-root MuiList-padding"
              >
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      1) Will not, under no circumstance, Combat Log.
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      2) Will not attack other USC Personnel or any allies of the United Systems Cooperative.
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      3) Will not attack any innocent CMDRs, unless in pre-arranged combat, or for Powerplay.
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      4) Will not, under no circumstance, discriminate against other commanders on the basis of race, religion, sexuality, nationality, etc.
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      5) Will not, under no circumstance, seek revenge against other USC Personnel, directly or indirectly. Punishments for crimes, and resolutions for transgressions, will be handed out by High Command.
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      6) Will not, under no circumstance, to divulge information marked as confidential by High Command.
                    </span>
                  </div>
                </li>
              </ul>
            </div>
            <div
              class="makeStyles-consequences-9"
            >
              <p
                class="MuiTypography-root MuiTypography-body1"
              >
                Those found in violation of any of these Discord or Group rules, or found attempting to bypass these rules in any way, will first be subject to a minimum of a warning, further offences will be subject to demotion, expulsion, and/or USC Kill-on-Sight status.
              </p>
            </div>
          </div>
        </div>
        <div
          class="makeStyles-setGrid-5"
        >
          <div>
            <h3
              class="MuiTypography-root makeStyles-header-11 MuiTypography-h3"
            >
              High Command
            </h3>
            <div
              class="MuiPaper-root makeStyles-rank-12 MuiPaper-elevation1 MuiPaper-rounded"
            >
              <ul
                class="MuiList-root MuiList-padding"
              >
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      NickG2002
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Goliath the Second
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      LOKioo42
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Pablok
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      PracticalHalo
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      RedFang Culph
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      UltraNeros
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Admiralfeb
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Akhaten
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Aurelius Publius
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Captain Smokeq
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Clever Ape
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      IM2D
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Nullyti
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Queen Eleanor
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Yuting9
                    </span>
                  </div>
                </li>
              </ul>
            </div>
          </div>
          <div>
            <h3
              class="MuiTypography-root makeStyles-header-13 MuiTypography-h3"
            >
              Allies
            </h3>
            <div
              class="MuiPaper-root makeStyles-allies-14 MuiPaper-elevation1 MuiPaper-rounded"
            >
              <ul
                class="MuiList-root MuiList-padding"
              >
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      1st Imperium Auxiliary Cohort
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Alba Rotam Private Military
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Arbor Caelum
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Black Universe Gazers
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Daily Routine Incorperated
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Explorers of Nabudis
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      The Galactic Archive
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Gilgamesh Corps
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Independence Separatist Fleet
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Intergalactic Astronomical Union
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Interstellar Incorporated
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Knights of Pleiades
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Ryders of the Void
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      ROOKS
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Sapere Aude
                    </span>
                  </div>
                </li>
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <div
                    class="MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      The Altruists
                    </span>
                  </div>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
